src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -79,10 +145,50 @@ export default class OrganizationIntegrations extends AsyncComponent {
             <Box px={2} flex="1">
               {t('Integrations')}
             </Box>
+            {this.state.reloading && <StyledLoadingIndicator mini />}
           </PanelHeader>
           <PanelBody>{integrations}</PanelBody>
         </Panel>
       </React.Fragment>
     );
   }
 }
+
+const StyledLoadingIndicator = styled(LoadingIndicator)`
+  position: absolute;
+  right: 7px;
+  top: 50%;
+  transform: translateY(-16px);
+`;
+
+const ProviderName = styled.div`
```

Comment:
I wonder if we could make a lint rule for this

Reply:
You're talking about wanting `styled('div')` right?

Reply:
Yeah, it's so insignificant that it should just be handled by eslint some time in the future

Reply:
Kind of annoying that styled even lets you do this in that case :/ It's not deprecated is it?

Reply:
Not deprecated, I just think it's better to be consistent so that people reading code don't get confused why there are two way to do things. I think it exists to match styled-components syntax